components/script/dom/debuggerglobalscope.rs

```diff
@@ -155,6 +166,28 @@ impl DebuggerGlobalScope {
             "Guaranteed by DebuggerAddDebuggeeEvent::new"
         );
     }
+
+    pub(crate) fn fire_get_possible_breakpoints(
+        &self,
+        can_gc: CanGc,
+        spidermonkey_id: u32,
+        result_sender: IpcSender<Vec<devtools_traits::RecommendedBreakpointLocation>>,
+    ) {
+        assert!(
+            self.get_possible_breakpoints_result_sender
+                .replace(Some(result_sender))
+                .is_none()
+        );
+        let event = DomRoot::upcast::<Event>(GetPossibleBreakpointsEvent::new(
+            self.upcast(),
+            spidermonkey_id,
+            can_gc,
+        ));
+        assert!(
+            DomRoot::upcast::<Event>(event).fire(self.upcast(), can_gc),
+            "Guaranteed by AddDebuggeeEvent::new"
```

Comment:
I don't think this is the right message.

Reply:
fixed in 79edffdce11ccc38ce11b5a8fc2b41c26ff82dff